### PR TITLE
chore: Add missing release.sh

### PR DESCRIPTION
**Pull request by rhuss**

### CODE REVIEW

**Comment by rhuss** [comment]
@maximilien I would need an approve here as this just add the release script that is required for doing the, well, release ;-)

Please approve via GitHub's "Approve" button when doing the review, the workflow has changed (so no "/lgtm" or "/approve" for sandbox projects)

**Review by maximilien — APPROVED** [review]
/lgtm

**Pull request merged**
